aws-sdk-s3-transfer-manager/src/error.rs

```diff
@@ -48,6 +48,10 @@ pub enum ErrorKind {
     /// The operation is being canceled because the user explicitly called `.abort` on the handle,
     /// or a child operation failed with the abort policy.
     OperationCancelled,
+
+    /// Request/response validation failed (e.g., the total number of UploadPart requests sent
+    /// did not match the expected number of parts)
+    ValidationFailed,
```

Comment:
This is probably ok though you could maybe roll this up into `ChunkFailed` if I squint.

Reply:
Updated in 4b7b9c1, just to avoid increasing the number of variants when we can.